vectoriadb-index/src/test/java/jetbrains/vectoriadb/index/L2PQKMeansTest.java

```diff
@@ -132,6 +131,105 @@ public void testPQKMeansQuality() throws Exception {
         }
     }
 
+    @Test
+    public void randomClusterInitializationVsKmeansPpBenchmark() throws Exception {
+        var vectors = loadSift10KVectors();
+        var clustersCount = 40;
+
+        var progressTracker = new NoOpProgressTracker();
+        try (var pqQuantizer = new L2PQQuantizer()) {
+            System.out.println("Generating PQ codes...");
+            pqQuantizer.generatePQCodes(SIFT_VECTOR_DIMENSIONS, 32, new ArrayVectorReader(vectors), progressTracker);
+
+            var count = 0;
+            var meanSilhCoeffDiff = 0f;
+            var meanSilhCoeffRandom = 0f;
+            var meanSilhCoeffPP = 0f;
+            for (var i = 0; i < 100; i += 1) {
+                var silhCoeffRandom = doPQMeansAndGetSilhouetteCoefficient(pqQuantizer, clustersCount, vectors, false, progressTracker);
+                var silhCoeffPP = doPQMeansAndGetSilhouetteCoefficient(pqQuantizer, clustersCount, vectors, true, progressTracker);
+
+                System.out.printf("silhouetteCoefficient random = %f%n", silhCoeffRandom);
+                System.out.printf("silhouetteCoefficient k-means++ = %f%n", silhCoeffPP);
+
+                meanSilhCoeffDiff += silhCoeffPP - silhCoeffRandom;
+                meanSilhCoeffRandom += silhCoeffRandom;
+                meanSilhCoeffPP += silhCoeffPP;
+                count++;
+            }
+            meanSilhCoeffDiff = meanSilhCoeffDiff / count;
+            meanSilhCoeffRandom = meanSilhCoeffRandom / count;
+            meanSilhCoeffPP = meanSilhCoeffPP / count;
+            System.out.printf("meanSilhCoeffDiff: %f, meanSilhCoeffRandom: %f, meanSilhCoeffPP: %f", meanSilhCoeffDiff, meanSilhCoeffRandom, meanSilhCoeffPP);
+        }
+    }
+
+    @Test
+    public void silhouetteVsSilhouetteFixedBenchmark() throws Exception {
```

Comment:
I am going to remove this benchmark, when we decide what silhouetteCoefficient implementation we keep.